gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/sql/execute/MemberLogsMessage.java

```diff
@@ -0,0 +1,392 @@
+package com.pivotal.gemfirexd.internal.engine.sql.execute;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.util.*;
+
+import com.gemstone.gemfire.DataSerializer;
+import com.gemstone.gemfire.cache.execute.ResultCollector;
+import com.gemstone.gemfire.distributed.DistributedMember;
+import com.gemstone.gemfire.distributed.internal.InternalDistributedSystem;
+import com.gemstone.gemfire.internal.cache.GemFireCacheImpl;
+import com.gemstone.gemfire.internal.lang.StringUtils;
+import com.google.common.io.ByteStreams;
+import com.pivotal.gemfirexd.internal.engine.distributed.message.GfxdFunctionMessage;
+import com.pivotal.gemfirexd.internal.engine.distributed.message.MemberExecutorMessage;
+import org.apache.log4j.Appender;
+import org.apache.log4j.FileAppender;
+import org.apache.log4j.Logger;
+
+
+public class MemberLogsMessage extends MemberExecutorMessage {
+
+  private Logger logger = Logger.getLogger(MemberLogsMessage.class);
+
+  private String memberId;
+  private Long offset;
+  //private int byteLength;
+  private Long byteLength;
+
```

Comment:
offset could be null so using Long type (Primitive data types cannot be null). 
byteLength can be changed to long.